README.md

```diff
@@ -10,11 +10,16 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versions
```

Comment:
I am not sure we can do that with the redirector.  I would have to sit and try to make that work before committing to it.